contracts/src/l1/TokenBridge.sol

```diff
@@ -305,33 +387,35 @@ contract TokenBridge is
         messagingContract.consumeMessageFromL2(l2TokenBridge, payload);
     }
 
-    function receiveRewards(uint256 l2sender, address recipient, uint256 amount) onlyValidL2Address(l2sender) external {
+    function receiveRewards(
+        uint256 l2sender,
+        address recipient,
+        uint256 amount
+    ) external onlyValidL2Address(l2sender) {
```

Comment:
Valid check is redundant here since the consume message will make sure it's correct

Reply:
good catch!